common/BUILD.gn

```diff
@@ -158,6 +159,12 @@ source_set("common") {
     "//ui/base",
   ]
 
+  if (toolkit_views) {
+    # This code has TOOLKIT_VIEWS ifdefs, but doesn't actually rely on anything
+    # else in views.
+    all_dependent_configs = [ "//ui/views:flags" ]
```

Comment:
I don't know if this is correct as it's the first time I've used the property, but it was necessary to ensure the `TOOLKIT_VIEWS` was defined before brave_features.cc is compiled.